### DISCUSSION TITLE
Add request size in Envoy API Gateway logs

### DISCUSSION THREAD

**Discussion by FabioNappi** [mia-platform/community#331] [Q&A] [answered]
### Which section?

Console

### Which is the version?

1.89.3

### Description

Hi, 

I need to log the size of the requests arriving to my Envoy API Gateway.
I'm making my first steps with this service thus I am asking for help on what configuration should I write.

Thanks.

**Comment by fredmaggiowski** [comment, marked as answer]
Hey, you can change the desired log format by setting the command line args from console, envoy provides documentation about supported formats here: https://www.envoyproxy.io/docs/envoy/latest/operations/cli.html#cmdoption-log-format

Is there any limitation within the Console you are noticing? 🤔

**Reply by FabioNappi** [reply]
The current line arg is `{"level":"%l","time":"%Y-%m-%dT%T.%fZ","scope":"%n","message":"%j"}`, but reading logs from console I don't recognize the structure. I'm not getting how they work.
Here is a (not prettified) log example:
```
{"remoteAddress":"34.91.89.131","requestURL":"/logo/download/64ef517a4b04580ab7b37c31.png","reqId":"a4b80c21-bab8-4c68-9597-2ef711870da5","requestMethod":"GET","remoteHostname":"miafintech-demo-fintech.cloud.azure.mia-fintech.io","responseSize":57,"requestProtocol":"HTTP/1.1","responseTime":0,"statusCode":404,"time":"2023-10-23T15:19:23.811Z","originalRequestPath":"/logo/download/64ef517a4b04580ab7b37c31.png","userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36"}
```

**Reply by theliuk** [reply]
Our envoy logs correctly:
our version: `envoyproxy/envoy:v1.24.1`
our cmd: 
```
-c
/etc/envoy/envoy.json
-l
{{LOG_LEVEL}}
--log-format
{"level":"%l","time":"%Y-%m-%dT%T.%fZ","scope":"%n","message":"%j"}
--drain-time-s
50
```

**Reply by FabioNappi** [reply]
I don't know what happen exactly before, but now logs have the expected structure.

Anyway the desired behavior in my case was to customize the value returned by the `%j` of the `message` field, and not to put something different from `%j`. Is it possible?
